BDArmory/Control/MissileFire.cs

```diff
@@ -7114,7 +7131,7 @@ public void SendTargetDataToMissile(MissileBase ml, bool clearHeat = true)
                 if (ml.targetVessel != null) Debug.Log($"[BDArmory.MissileData]: targetInfo sent for {ml.targetVessel.Vessel.GetName()}");
             }
             if (BDArmorySettings.DEBUG_MISSILES)
-                Debug.Log($"[BDArmory.MissileData]: firing missile at {(currentTarget != null && currentTarget.Vessel != null ? currentTarget.Vessel.GetName() : "null target")}");
+                Debug.Log($"[BDArmory.MissileData]: firing missile at {(targetVessel != null ? currentTarget.Vessel.GetName() : "null target")}");
```

Comment:
`currentTarget.Vessel.GetName()` should also be changed to `targetVessel.GetName()` if this change is intended, otherwise you're likely to get a NRE here.